src/controllers/defiPositions/defiPositions.ts

```diff
@@ -130,22 +130,25 @@ export class DefiPositionsController extends EventEmitter {
       this.#providers.providers
     )
 
-    this.emitUpdate()
     await this.#storage.set(
       'networksWithPositionsByAccounts',
       this.#networksWithPositionsByAccounts
     )
   }
 
-  async updatePositions(opts?: { chainId?: bigint; maxDataAgeMs?: number; forceUpdate?: boolean }) {
-    const { chainId, maxDataAgeMs, forceUpdate } = opts || {}
+  async updatePositions(opts?: {
+    chainIds?: bigint[]
+    maxDataAgeMs?: number
+    forceUpdate?: boolean
+  }) {
+    const { chainIds, maxDataAgeMs, forceUpdate } = opts || {}
     const selectedAccount = this.#selectedAccount.account
     if (!selectedAccount) return
 
     const selectedAccountAddr = selectedAccount.addr
-    const networksToUpdate = chainId
-      ? this.#networks.networks.filter((n) => n.chainId === chainId)
-      : this.#networks.networks
+    const networksToUpdate = chainIds
+      ? this.#networks.allNetworks.filter((n) => chainIds.includes(n.chainId))
+      : this.#networks.allNetworks
```

Comment:
It's a bit weird that we display defi positions for disabled networks (especially so because they aren't included in the total balance)
<img width="752" height="747" alt="image" src="https://github.com/user-attachments/assets/c0c3b0f4-e905-42b6-aa81-a26ddc9eee84" />